pitch.md

```diff
@@ -23,38 +51,192 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+a readable and writable language. I actually think this DSL addresses the desire to create art and 
+animation very effectively, and I wish I had known more about this before! I suppose a drawback of 
+the language is that it has such a wide range of features that it might require some investment of 
+time to become proficient with the language, and it is not necessarily the most intuitive for those 
+who have never programmed before.
+
+The [CSS3 Animations](http://www.w3schools.com/css/css3_animations.asp) library “allows animations of most HTML elements.” It is less expressive than Processing in the sense that it can only be used to create really basic animations or animate existing elements. I think a drawback of this is that it also requires knowledge of HTML and may be intimidating for people without programming experience. However, the syntax is pretty readable and I could see how I can use elements of it in my own DSL.
+
+Finally, there is a feature in Powerpoint that allows for [line animations](https://www.screenr.com/6Rg), and if we’re playing 
+the “Is this a DSL?” game, I definitely think this would count as a DSL. Animations in powerpoint 
+are really simple to use, and they allow people without programming knowledge to add animations in
+their presentations. This is an example of a 2D line animation that was created in Powerpoint!
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+I’d say 70% of the time will be spent on engaging in the language aspects. A lot of decisions 
+will need to be made on how much to limit the expressiveness of the language, because there are 
+an endless number of features that could be made available to the user. I think the challenge will
+be sticking to an even more specific domain within 2D line animations, so that the language will be 
+super easy and enjoyable to use. The other 30% of the time will be spent on the “systems” aspect, and 
+it will mostly be figuring out how to get the code translated to the GUI window where the animations 
+will be displayed. I will need to figure out the limitations of the window size and how many animations
+can simultaneously be executed.
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
 
+I plan to begin with implementing the only the bare bones features, which is allowing the user to define 
+a rule that specifies the coordinate where a line starts, the direction it travels towards, and the distance
+it travels. I envision this DSL being implemented as an external DSL in Scala, although I’d like to look more
+into other possible host languages. I’d say that these features wouldn’t take too long to design and implement. 
```

Comment:
Maybe look into other programming languages that can do GUI stuff easily since the ideal output would be a video/animation of the 2d drawing.